test/shared/src/main/scala/zio/test/TestAspect.scala

```diff
@@ -404,39 +404,46 @@ object TestAspect extends TimeoutVariants {
   def parallelN(n: Int): TestAspectPoly = executionStrategy(ExecutionStrategy.ParallelN(n))
 
   /**
-   * An aspect that restores a given [[Restorable]]'s state to its starting state after the test is run.
+   * An aspect that restores a given [[zio.test.environment.Restorable Restorable]]'s state to its starting state
```

Comment:
Do we need the second `Restorable` here? I thought this would be written as "An aspect that restores a given [[zio.test.environment.Restorable]]'s state to its start state after the test is run.", but I could be wrong. Similar question for the changes below.

Reply:
[[zio.test.environment.Restorable]] is rendered as `zio.test.environment.Restorable` in the scaladoc html, [[zio.test.environment.Restorable Restorable]] is rendered as `Restorable`, which makes the text flow a bit better

Reply:
Ah, great.